expr/http_endpoint.go

```diff
@@ -423,8 +391,6 @@ func (e *HTTPEndpointExpr) Validate() error {
 				if r.Body != nil && r.Body.Type == Empty {
 					verr.Add(r, "Response body empty but endpoint defines streaming WebSocket response.")
 				}
-			} else if successResp && e.SkipResponseBodyEncodeDecode {
```

Comment:
The idea with this test was that it's not possible for Goa to know what response status to generate if there is no result to the service level function (and therefore no field to compare with the `Tag` value defined in the design). So maybe the test should also make sure that there is only a body defined (in which case there won't be a result) but in that case it seems to be still needed?